.github/workflows/notstale.yml

```diff
@@ -0,0 +1,43 @@
+# This workflow adds not stale label to only copybara[bot] PRs to avoid stale workflow 
+#
+# You can adjust the behavior by modifying this file.
+# For more information, see:
+# https://github.com/actions-ecosystem/action-add-labels
+name: Label Not Stale PRs
+
+on:
+  pull_request:
+    types: [opened, synchronize]
+
+jobs:
+  label-not-stale-prs:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check PR author
+        id: pr-author-check
+        uses: actions/github-script@v4
+        with:
+          github-token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
@keertk I think we will have the same problem providing the github-token here, or is this different?